k8s/init-services/11-keycloak.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: keycloak
+  name: keycloak
+  namespace: ersms-forum
+spec:
+  ports:
+    - name: "http"
+      port: 8888
+      targetPort: 8080
+  selector:
+    app: keycloak
+  type: LoadBalancer # TODO: unsure?
```

Comment:
This depends whether we need to expose the keycloak outside of the cluster and assign a port to it.

Imo if only our services communicate with the keycloak we could remove the type and it will default to ClusterIP:
- <https://www.baeldung.com/ops/kubernetes-service-types>
- <https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service>

When set to ClusterIP we still can access the service via port-forwarding. Here we've an example usage:
- <https://github.com/ERSMS-24L/ERSMS-24L/blob/master/deploy-to-kubernetes.sh#L30>
```sh
kubectl port-forward svc/prometheus-stack-kube-prom-prometheus 9090:9090
```

Or we can setup the route via the gateway service and assign only a path like `/keycloak` to proxy the service.

Reply:
Yes, FE needs to access it. I think I will route it through the gateway, but that will be in a separate PR.